src/cdk/a11y/focus-monitor/focus-monitor.ts

```diff
@@ -493,7 +520,7 @@ export class FocusMonitor implements OnDestroy {
 }
 
 /** Gets the target of an event, accounting for Shadow DOM. */
-function getTarget(event: Event): HTMLElement|null {
+export function getTarget(event: Event): HTMLElement|null {
```

Comment:
With this change, this file and the `input-modality-detector.ts` file both import from one another. I'd like to avoid this if possible. Should I move this small helper to a separate file? If so, where's a good place? @crisbeto

Reply:
I ended up moving `getTarget` to `input-modality-detector.ts` to resolve the circular dependencies (apparently there's some circular deps golden check), but it still doesn't feel like the right place for this small helper to live.